pipeline.py

```diff
@@ -91,7 +103,37 @@ def validate_dataset(path_to_validate):
     """
     Validates folder with assets
     """
-    pass
+    path_to_validate = Path(path_to_validate)
+
+    files = [file for file in path_to_validate.glob('*')]
+    stems = [file.stem for file in files]
+    raw_txt = 0
+    meta_json = 0
+
+    if not path_to_validate.exists():
+        raise FileNotFoundError
+
+    if not path_to_validate.is_dir():
+        raise NotADirectoryError
+
+    if not files:
+        raise EmptyDirectoryError
+
+    for file in files:
+        if file.suffix == '.json':
```

Comment:
I completely did not get this check: you have doubled checks and they look very strange. think again what exactly you want to achieve and propose a better algorithm for that